scripts/clean_up_images.py

```diff
@@ -0,0 +1,36 @@
+import logging
+import subprocess
+
+LIST_IMAGES_COMMAND_PREFIX = 'gcloud container images list-tags'
+DELETE_IMAGES_COMMAND_PREFIX = 'gcloud container images delete'
+
+# Allow at most 3 versions of images stored at container registry
+ALLOWED_SIZE = 3
+
+
+def delete_old_images(image_name):
+    list_command = LIST_IMAGES_COMMAND_PREFIX + \
+        ' gcr.io/python-compatibility-tools/{}'.format(image_name) + ' --format=get(digest)'
+    digests = subprocess.check_output(list_command.split(' '))
+    digests_list = digests.decode('utf-8').strip('\n').split('\n')
+    num_of_digests = len(digests_list)
+
+    if num_of_digests > ALLOWED_SIZE:
+        to_delete = digests_list[ALLOWED_SIZE:]
+        print(to_delete)
+        for digest in to_delete:
+            command = DELETE_IMAGES_COMMAND_PREFIX + \
+                ' gcr.io/python-compatibility-tools/{}@{}'.format(image_name, digest) \
+                + ' --force-delete-tags'
```

Comment:
It might be a good idea to ask for confirmation after displaying the images that will be deleted before executing the command.

Reply:
Thanks for reviewing! I'm going to close this PR as #243 is checked in.